core/src/tx.rs

```diff
@@ -120,11 +120,6 @@ impl AcceptedTransaction {
         Ok(Self(tx))
     }
 
-    #[inline]
-    pub(crate) fn merge_signatures(&mut self, other: Self) -> bool {
-        self.0.merge_signatures(other.0)
-    }
```

Comment:
Looking at this removed code I remembered how much thing in Iroha were just *prior optimizations* which becomed useless...

no_std, expressions, query predicates and now mst